addon/__test__/options.test.jsx

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable */
+const esmImport = require("esm")(module, { cjs: false, force: true });
+const { act } = require("react-dom/test-utils");
+const enzyme = require("enzyme");
+const Adapter = require("enzyme-adapter-react-16");
+
+enzyme.configure({ adapter: new Adapter() });
+
+// Browser code expects window to be the global object
+global.window = global.globalThis = global;
+// We need to make a global nodeRequire function so that our module
+// loading will use native node module loading instead of the default.
+global.nodeRequire = require;
+
+// Mock `fetch`, which is used to get localization info when running in the browser
+global.fetch = function(url) {
+  const fileSystem = require("fs");
+  const path = require("path");
+  const ROOT_PATH = path.join(__dirname, "..");
+  const filePath = path.join(ROOT_PATH, url);
+
+  const data = fileSystem.readFileSync(filePath, "utf8");
+  return Promise.resolve({
+    json: function() {
+      return Promise.resolve(JSON.parse(data));
+    },
+  });
+};
```

Comment:
Just a thought: is there any way we can avoid the global setup duplication for each test file?

Are we really needing React to be able to be loaded as es6 files?

Reply:
I can move these to an import that is imported to all tests. Since there was only one test at this point, I put them at the top.